hazelcast/src/test/java/com/hazelcast/internal/metrics/impl/MetricsCompressorTest.java

```diff
@@ -51,49 +50,49 @@
 public class MetricsCompressorTest {
 
     private static final String LONG_NAME = Stream.generate(() -> "a")
-                                                 .limit(MetricsDictionary.MAX_WORD_LENGTH + 1)
-                                                 .collect(Collectors.joining());
+            .limit(MetricsDictionary.MAX_WORD_LENGTH + 1)
+            .collect(Collectors.joining());
 
     private final DefaultMetricDescriptorSupplier supplier = new DefaultMetricDescriptorSupplier();
-    private final Supplier<? extends MetricDescriptor> supplierSpy = spy(supplier);
+    private final SpiedMetricDescriptor spiedSupplier = new SpiedMetricDescriptor(supplier);
     private final MetricsCompressor compressor = new MetricsCompressor();
 
     @Test
     public void testSingleLongMetric() {
         MetricDescriptor originalMetric = supplier.get()
-                                                  .withPrefix("prefix")
```

Comment:
This is not re-formatting on my machine. Are you sure you have styles imported?

Reply:
> This is not re-formatting on my machine. Are you sure you have styles imported?

Turns out at some point I managed to reset them! Thanks for the catch, sorted now 👍